locales-src/en-US/strings.properties

```diff
@@ -213,3 +213,7 @@ firstrun_skip_login=Skip this step
 
 # LOCALIZATION NOTE (context_menu_title): Action tooltip to open a context menu
 context_menu_title=Open menu
+
+# This message is shown to nightly en-US users viewing New Tab experience in dark mode.
+ds_message_content=This version of New Tab doesn not support dark mode yet.
```

Comment:
does not, doesn't? I don't see mention of copy in the bug.

Also, are we showing it really only to en-US users?

Reply:
We'll show to all users but because this is temporary, I believe it should be fine to not bother localizing. Hopefully we get dark mode in before the end of 67 nightly.

Reply:
> We'll show to all users but because this is temporary, I believe it should be fine to not bother localizing. Hopefully we get dark mode in before the end of 67 nightly.

If that's the plan it should be hardcoded. If you put it in the .properties file it will get translated.